docs/LISTING-README-TEMPLATE.md

```diff
@@ -0,0 +1,97 @@
+# Description
+**Focus on what this software is and the VALUE it brings to users.**
+------
+(replace all text in italics.)
+
+_Write a few paragraphs about *WHAT* the listing is, *WHAT* the software does, and the *VALUE* it brings to users._
+
+_Include specific details on what this 1-click provides, and the value to users. Enumerate the values this 1-click's config gives the user. Enumerate the value this software combination, specifically for stacks containing multiple pieces of software, bring to the user. 
```

Comment:
italics broke here, minor

Reply:
Thanks for the review.
Good catch. I'll correct.

Reply:
Corrected, and updated.